agent_deploy/kubernetes/sysdig-agent-daemonset-v1.yaml

```diff
@@ -18,6 +18,9 @@ spec:
         app: sysdig-agent
     spec:
       volumes:
+      - name: modprobe-d
+        hostPath:
+          path: /etc/modprobe.d
```

Comment:
Is this path guaranteed to exist on all hosts? What happens if it doesn't exist?

Reply:
That's a standard directory, so I'd be surprised to find a host that doesn't have it.  If such a host existed, then it'd probably create an empty directory on the host at that path and mount in that empty directory into the container.